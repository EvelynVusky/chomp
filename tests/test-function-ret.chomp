int a() {
    return 1;
} 

string s() {
    return "hello";
} 

bit b() {
    return {{0}};
} 
nibble n() {
    return {{0101}};
} 
char c() {
    return 'a';
} 
bool bo() {
    return false;
} 
int main() {
    println(a());
    println(s());
    println(b());
    println(n());
    println(c());
    println(bo());
    return 0;
}