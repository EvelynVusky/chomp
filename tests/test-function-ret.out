1
hello
0
0101
a
0
